nvtabular/ops/target_encoding.py

```diff
@@ -94,6 +93,8 @@ class TargetEncoding(StatOperator):
         while low-cardinality columns can likely use `tree_width=1`.
         If passing a dict, each key and value should correspond to the column
         name and width, respectively. The default value is 8 for all columns.
+    cat_cache : ToDo Describe
+        Text
```

Comment:
Lets take this from categorify:
```suggestion
    cat_cache : {"device", "host", "disk"} or dict
        Location to cache the list of unique categories for
        each categorical column. If passing a dict, each key and value
        should correspond to the column name and location, respectively.
        Default is "host" for all columns.
```